src/Common/MoistThermodynamics/isentropic.jl

```diff
@@ -0,0 +1,44 @@
+export IsentropicProcess
+
+export air_pressure_given_θ, air_pressure, air_temperature
+
+"""
+    IsentropicProcess
+
+For dispatching to isentropic formulas
+"""
+struct IsentropicProcess end
+
+"""
+    air_pressure_given_θ(θ::FT, Φ::FT, ::IsentropicProcess)
+
+The air pressure for an isentropic process, where
+
+ - `θ` potential temperature
+ - `Φ` gravitation potential
+"""
+air_pressure_given_θ(θ::FT, Φ::FT, ::IsentropicProcess) where {FT} =
+  FT(MSLP) * (1 - Φ / (θ * FT(cp_d))) ^ (FT(cp_d) / FT(R_d))
```

Comment:
Consider using moist specific heats, gas constants etc. here and elsewhere?

Reply:
Wouldn't using moist specific heats contradict `DryAdiabaticProcess`?

Reply:
By ‘dry’ we usually mean unsaturated and non-condensing (as in an ocean is dry :). If you want to be more specific, you can say ‘unsaturated’.